### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jayfurmanek**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [x] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

Add some more details to the README

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ test/
 * `conda-build` >= 3.20
   * Once `conda` is installed, `conda-build` can be installed with the command: `conda install conda-build`
 * `python` >= 3.6
+* `docker` >= 1.13
+  * Docker is only required when using the `--docker_build`` option (see below).
```
Comment (bethune-bryant) [inline README.md:58]:
```suggestion
  * Docker is only required when using the `--docker_build` option (see below).
```

File: README.md [hunk 2]
```diff
@@ -66,17 +68,27 @@ When building packages that use CUDA, a tar package of TensorRT 7.0 for CUDA 10.
 
 ### Building a Collection of Packages
 
-The `build_env.py` script can be used to build a collection of Open-CE packages. An environment file needs to be passed in as input. A selection of environment files are provided within the `envs` directory for different frameworks such as TensorFlow and PyTorch. The output from running `build_env.py` will be a local conda channel (by default called `condabuild`). For more details on `build_env.py`, please see [`doc/README.build_env.md`](doc/README.build_env.md).
+To build an entire integrated and function conda channel using Open-CE, start by installing the needed tools in the [Requirements](#requirements) section above.
```
Comment (bethune-bryant) [inline README.md:71]:
```suggestion
To build an entire integrated and functional conda channel using Open-CE, start by installing the needed tools in the [Requirements](#requirements) section above.
```

File: README.md [hunk 3]
```diff
@@ -66,17 +68,27 @@ When building packages that use CUDA, a tar package of TensorRT 7.0 for CUDA 10.
...
+The `build_env.py` script can then be used to build a collection of Open-CE packages. An Open-CE environment file needs to be passed in as input. A selection of environment files are provided within the `envs` directory for different frameworks such as TensorFlow and PyTorch. The output from running `build_env.py` will be a local conda channel (by default called `condabuild`). For more details on `build_env.py`, please see [`doc/README.build_env.md`](doc/README.build_env.md).
 
-The following commands will build all of the Open-CE packages for Python 3.6, including CUDA builds and cpu-only builds. The commands should be run from within the same directory that contains `local_files`.
+The following commands will use the opence-env.yaml Open-CE environment file to build all of the Open-CE packages for Python 3.6 (the default), including CUDA builds and cpu-only builds (also the default). The commands should be run from within the same directory that contains `local_files`.
 
 ```bash
-# Clone Open-CE from GitHub
-git clone https://github.com/open-ce/open-ce.git
+# Clone a specific Open-CE release from GitHub
+git clone https://github.com/open-ce/open-ce.git --branch open-ce-v1.0.0
```
Comment (bnemanich) [inline README.md:78]: Can we only have the specific branch clone in the second example? I would prefer the first one to be as simple as possible. When describing the specific comment, we should add something like `# Clone a specific Open-CE release from GitHub (in this case version 1.0.0)`.

Comment (jayfurmanek) [reply README.md:78]: Sure. Thats a good idea.

**Review by bethune-bryant — APPROVED** [review]
Looks good!

**Pull request merged**